place_bot/place_bot.py

```diff
@@ -160,14 +160,15 @@ def _find_differing_pixels(map_data, im_data, shape):
         return differing_pixels
 
     @staticmethod
-    def image_to_data(image: Image, shape: Tuple[int, int], indexed_color=False):
+    def image_to_data(image: Image, shape: Tuple[int, int], indexed_color=False, fuzzy=False):
         if indexed_color:
             data = np.array(image.getdata())
 
             # all color indices are off by 1
             data = data - 1
         else:
-            data = np.array([Color.from_pixel(px).value.id for px in image.getdata()])
+            from_pixel = Color.from_pixel if not fuzzy else Color.closest_from_pixel
```

Comment:
Rather than have two options, fuzzy and not fuzzy, would it make sense to just always assume fuzzy is desired? Like, `closest_from_pixel` could first check to see if there's an exact match, and return it if there is, and otherwise just try to find the closest match. What are your thoughts on this?

Reply:
Good idea :+1: Done